From ed4923725aa72850a06b5273653677a1dbcea35f Mon Sep 17 00:00:00 2001
From: Suman Anna <s-anna@ti.com>
Date: Mon, 19 May 2014 17:15:03 -0500
Subject: [PATCH 1137/1587] ARM: dts: AM33xx: Add PRU mboxes to mailbox node

The mailbox DT node data is updated with additional mbox data to
communicate with the PRU0 (User Id 1) and PRU1 (User Id 2) cores
within the PRU-ICSS subsystem. The following are the assignments
of mailbox fifos between MPU and the remote PRU cores:

		MPU Tx/PRU Rx	MPU Rx/PRU Tx
	PRU0	2		3
	PRU1	4		5

Signed-off-by: Suman Anna <s-anna@ti.com>
[zou: Original patch taken from
ti-sdk-am335x-evm-08.00.00.00-Linux-x86-Install.bin]
Signed-off-by: Cao Zou <cao.zou@windriver.com>
---
 arch/arm/boot/dts/am33xx.dtsi |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/am33xx.dtsi b/arch/arm/boot/dts/am33xx.dtsi
index b4bfc05..46d730a 100644
--- a/arch/arm/boot/dts/am33xx.dtsi
+++ b/arch/arm/boot/dts/am33xx.dtsi
@@ -347,6 +347,14 @@
 				ti,mbox-tx = <0 0 0>;
 				ti,mbox-rx = <0 0 3>;
 			};
+			mbox_pru0: mbox_pru0 {
+				ti,mbox-tx = <2 0 0>;
+				ti,mbox-rx = <3 0 0>;
+			};
+			mbox_pru1: mbox_pru1 {
+				ti,mbox-tx = <4 0 0>;
+				ti,mbox-rx = <5 0 0>;
+			};
 		};
 
 		timer1: timer@44e31000 {
-- 
1.7.5.4

